src/module/so-inspired.js

```diff
@@ -139,6 +204,6 @@ function removeInspiration(user, _sheet) {
 
 function updateSheetForInspo(user) {
   if (user?.character.sheet.rendered) {
-    user.character.sheet.render(true);
+    user.character.sheet.render(false);
```

Comment:
@mysurvive this is mostly just a recommendation.
`render(true)` can have some surprising effects when it fires off for each user change.
Tidy and the new Default Sheets can frequently have user changes that do not relate to this module which can be triggered from actions within the sheet, such as changing the sort on various tabs.
I tested it and noted that the legacy sheet and Tidy were refreshing successfully with a `render(false)`.